test/libsolidity/smtCheckerTests/operators/tuple_rationals_conditional.sol

```diff
@@ -0,0 +1,9 @@
+pragma experimental SMTChecker;
+
+contract C {
+	function f(bool x) public pure {
+		(uint a, uint b) = x ? (10000000001, 2) : (3, 4);
+		assert(a % 2 == 1);
+		assert(b % 2 == 0);
+	}
```

Comment:
I'd also assert for `a != 0 && b != 0` to avoid any potential case where the tuple is not considered initialised.

Reply:
Done